docs/tutorials/t2hahn_characterization.ipynb

```diff
@@ -0,0 +1,590 @@
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The first circuirs of hahn echo experiment with 0 echoes:\n",
+      "     ┌─────────┐┌─────────────────┐┌──────────┐┌─┐\n",
+      "  q: ┤ Rx(π/2) ├┤ Delay(1e-06[s]) ├┤ Rx(-π/2) ├┤M├\n",
+      "     └─────────┘└─────────────────┘└──────────┘└╥┘\n",
+      "c: 1/═══════════════════════════════════════════╩═\n",
+      "                                                0 \n",
+      "The first circuirs of hahn echo experiment with 4 echoes:\n",
+      "     ┌─────────┐┌─────────────────┐┌───────┐┌─────────────────┐»\n",
+      "  q: ┤ Rx(π/2) ├┤ Delay(1e-06[s]) ├┤ Rx(π) ├┤ Delay(1e-06[s]) ├»\n",
+      "     └─────────┘└─────────────────┘└───────┘└─────────────────┘»\n",
+      "c: 1/══════════════════════════════════════════════════════════»\n",
+      "                                                               »\n",
+      "«     ┌─────────────────┐┌───────┐┌─────────────────┐┌─────────────────┐»\n",
+      "«  q: ┤ Delay(1e-06[s]) ├┤ Rx(π) ├┤ Delay(1e-06[s]) ├┤ Delay(1e-06[s]) ├»\n",
+      "«     └─────────────────┘└───────┘└─────────────────┘└─────────────────┘»\n",
+      "«c: 1/══════════════════════════════════════════════════════════════════»\n",
+      "«                                                                       »\n",
+      "«     ┌───────┐┌─────────────────┐┌─────────────────┐┌───────┐»\n",
+      "«  q: ┤ Rx(π) ├┤ Delay(1e-06[s]) ├┤ Delay(1e-06[s]) ├┤ Rx(π) ├»\n",
+      "«     └───────┘└─────────────────┘└─────────────────┘└───────┘»\n",
+      "«c: 1/════════════════════════════════════════════════════════»\n",
+      "«                                                             »\n",
+      "«     ┌─────────────────┐┌──────────┐┌─┐\n",
+      "«  q: ┤ Delay(1e-06[s]) ├┤ Rx(-π/2) ├┤M├\n",
+      "«     └─────────────────┘└──────────┘└╥┘\n",
+      "«c: 1/════════════════════════════════╩═\n",
+      "«                                     0 \n"
+     ]
+    }
+   ],
+   "source": [
+    "import numpy as np\n",
+    "\n",
+    "qubit2 = 0\n",
+    "# set the desired delays\n",
+    "conversion_factor = 1e-6\n",
+    "\n",
+    "delays2 = np.append(\n",
+    "                    (np.linspace(1.0, 10.0, num=37)).astype(float),\n",
+    "                    (np.linspace(10.5, 45.0, num=70)).astype(float),\n",
+    "                )\n",
+    "delays2 = np.append(\n",
+    "                    delays2,\n",
+    "                    (np.linspace(45.5, 200, num=140)).astype(float),\n",
+    "                )\n",
+    "\n",
+    "delays2 = [float(_) * conversion_factor for _ in delays2]\n",
+    "\n",
+    "# Delays for the 0 echo circuit\n",
+    "delays3 = np.append(\n",
+    "                    (np.linspace(1.0, 10.0, num=110)).astype(float),\n",
+    "                    (np.linspace(10.5, 25.0, num=137)).astype(float),\n",
+    "                )\n",
+    "delays3 = [float(_) * conversion_factor for _ in delays3]\n",
```

Comment:
Added comment :
```
# The delays aren't equally sparse due the behavior of exponential decay curve where the change in the result
# in earlier times is bigger then later times. In addition, since the delay amount is  'delay * 2 * num_of_echoes',
# the construction of the delays for each experiment will be different so they will have matched total length.
```